### ISSUE TITLE
site not loading on Firefox mobile Android 

### ISSUE THREAD

**Issue by peter-kal** [cogentcore/cogentcore.github.io#1]
First of all, congrats for your project's initial release, second of all, when opening the website in Firefox mobile Android, it takes ages for the progress bar to reach something like 50%, and then the whole site reboots and starts all over again, needed to open from chrome.

**Comment by kkoreilly** [comment]
Thank you for reporting this! I cannot reproduce this issue in Firefox on iOS; I will test it on an Android device soon.

**Comment by kkoreilly** [comment]
We also cannot reproduce this issue in Firefox on Android. Can you try again and see if the issue still happens with our new changes? Does it happen reliably every time?

**Comment by peter-kal** [comment]
Yep, it still doesn't load, sometimes the progress bar reaches 99% and the restarts. after a few it crashes firefox as a whole. I tried recording it, and when it crashed firefox it also stopped the recording.

**Comment by kkoreilly** [comment]
This issue will probably be fixed when we switch to GopherJS, which we are planning to do soon (https://github.com/cogentcore/core/issues/974). In the meantime, you can use Chrome. Apologies for the inconvenience.

**Comment by peter-kal** [comment]
No worries, wish had more know-how and time to contribute 👍!
